jhu-idc/idc-isle-dc
# Use composer 2

Upgrades composer to 2.x.  Includes minor changes composer 2 makes to lockfile.

## To test
* Run the stack in the PR, then hop into the Drupal container via `docker exec -it idc_drupal_1 ash`
* Do `composer --version`, you should see `Composer version 2.1.6 2021-08-19 17:11:08`
* Feel free to play around with `composer` by upgrading or installing a module

Resolves #218
